app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    @user = User.find_by id: params[:followed_id]
```

Comment:
plz, handling when `@user` is nil 
change by steps
 + remove this line
 + add method into this file
`private`
`def load_user`
` @user = User.find_by id: params[:followed_id]`
`return if @user`
`flash[:warning] = "User not found"`
`redirect_to root_path` 

+ then use callback for action create
add code `before_action :load_user, only: :create`